easybuild/easyblocks/p/pytorch.py

```diff
@@ -121,6 +121,19 @@ def configure_step(self):
         """Custom configure procedure for PyTorch."""
         super(EB_PyTorch, self).configure_step()
 
+        # PyTorch preferes cmake3 over cmake which usually does not exist
+        cmake_root = get_software_root('CMake')
+        cmake_version = get_software_version('CMake')
+        if cmake_root and not os.path.isfile(os.path.join(cmake_root, 'bin', 'cmake3')):
+            if cmake_version and cmake_version.split('.')[0] != '3':
+                raise EasyBuildError('PyTorch requires CMake 3 but CMake %s was found', cmake_version)
+            cmake_bin_dir = tempfile.mkdtemp(suffix='cmake-bin')
+            self.log.warning('Creating symlink `cmake3` in %s to avoid PyTorch picking up a system CMake. ' +
+                             'Reinstall the CMake module to avoid this!', cmake_bin_dir)
+            os.symlink(os.path.join(cmake_root, 'bin', 'cmake'), os.path.join(cmake_bin_dir, 'cmake3'))
```

Comment:
Same here, please use `symlink` function provided by `filetools` module.

Also, this actually belong in `prepare_step`, where we set up the build environment?